syntax = "proto2";
package Devialet.IMASlave4U;
import "CallMeMaybe/CallMeMaybe.proto";
option py_generic_services = true;

service SoundDesign {
	option (Devialet.CallMeMaybe.dvltServiceOptions) = {
		serviceName: "com.devialet.twerkit.sounddesign-0.imaslave4u-0"
		baseService: "Devialet.TwerkIt.SoundDesign"
		
	};
}